Documentation/internals/cilium_operator.rst

```diff
@@ -77,6 +85,12 @@ mode:
 
 For more information on IPAM visit :ref:`address_management`.
 
+LB IPAM
+~~~~~~~
```

Comment:
Readability nit:

```suggestion
Load Balancer IP Address Management
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
Thanks for your suggestion! Fixed in the latest push.